src/Sarif.Converters/SemmleQlConverter.cs

```diff
@@ -14,9 +14,9 @@ namespace Microsoft.CodeAnalysis.Sarif.Converters
     /// <summary>
     /// Converts a log file from the Semmle format to the SARIF format.
     /// </summary>
-    internal class SemmleQLConverter : ToolFileConverterBase
+    public class SemmleQLConverter : ToolFileConverterBase
```

Comment:
You can't refer to a non-public type that's a generic constraint in a public class.  MSTEST requires a test class to be public. And so I had to 'publicize' the transitive closure of types implied by 

MyTest<MyProductType>

---
In reply to: [134351055](https://github.com/Microsoft/sarif-sdk/pull/590#discussion_r134351055) [](ancestors = 134351055)